pandas/core/arrays/integer.py

```diff
@@ -377,14 +377,28 @@ def __getitem__(self, item):
             return self._data[item]
         return type(self)(self._data[item], self._mask[item])
 
-    def _coerce_to_ndarray(self):
+    def _coerce_to_ndarray(self, dtype=None, na_value=libmissing.NA):
```

Comment:
I think  we'll want to make a `to_array` that's basically this method.

Reply:
we already have ``to_numeric`` which is the canonical form of ``to_array`` (rather do conversions there)

Reply:
Jeff, the discussion about `to_numpy` (the `to_array` was a typo I think) moved to https://github.com/pandas-dev/pandas/issues/30038 in the mean time. Can you move your comment there if relevant?

Note that `to_numeric` is a function that converts any thing to a numeric type. While this function here is to convert a numeric type (this IntegerArray) to any other numpy dtype.